radars/SRE.csv

```diff
@@ -0,0 +1,951 @@
+<h3>Pros</h3>
+<ul>
+<li>Tight integration with Visual Studio</li>
+<li>Integrated project management and development experience</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Centralized source control</li>
+<li>Gated check-ins painful</li>
+<li>Slow interfaces</li>
+</ul>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Subversion,Infrastructure,Divest,none,"<p><a href=""https://subversion.apache.org/"">Subversion</a> is an open-source, centralized version control system characterized by its reliability as a safe haven for valuable data; the simplicity of its model and usage; and its ability to support the needs of a wide variety of users and projects, from individuals to large-scale enterprise operations.</p>
+<h3>Pros</h3>
+<ul>
+<li>Stable</li>
+<li>Well-known</li>
+<li>Simple rollbacks</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Lack of workflows (pull requests)</li>
+<li>Centralized source control</li>
+<li>Not as friendly to .NET developers</li>
+<li>Refactoring painful</li>
+</ul>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+Stackify,Infrastructure,Divest,none,"<p><a href=""https://stackify.com/"">Stackify Retrace</a> combines APM, centralized logging, code profiling, integrated errors and logs, and more under one roof. That way, everyone is in the know.</p>
+<h3>Pros</h3>
+<ul>
+<li>Combined code profiler with application logging</li>
+<li>Easy to deploy</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Weak mobile support</li>
+<li>Documentation needs improvement</li>
+<li>Cost</li>
+<li>Not as highly rated as other APM providers</li>
+</ul>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+AppDynamics,Infrastructure,Divest,none,"<p><a href=""https://www.appdynamics.com"">AppDynamics</a> is used to actively monitor, analyze and optimize complex application environments at scale.</p>
+<h3>Pros</h3>
+<ul>
+<li>Extensive displays and dashboards</li>
+<li>Efficient agents with minimal overhead</li>
+<li>Agentless deployment</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Higher learning curve than others</li>
+<li>More expensive than other vendors</li>
+<li>Complex interface</li>
+</ul>
+<p>Replaced by <a href="""">NewRelic</a>.</p>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2021</p>
+"
+AMI (Custom built),Infrastructure,Divest,none,"<p><a href=""https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instances-and-amis.html"">AMI (Custom Built)</a> is a template that contains a software configuration (for example, an operating system, an application server, and applications). From an AMI, you launch an instance, which is a copy of the AMI running as a virtual server in the cloud.</p>
```

Comment:
I'm not sure I agree with this.  I suspect we use (or could/should use) this in our hand-rolled K8S cluster and it could also be used in EKS, I think.  That said, I'm not positive that we want to use it as part of an EC2 strategy.

Reply:
Switched from divest to hold. Will need follow on conversations to resolve this.